identity/app/views/profile/emailSettings.scala.html

```diff
@@ -27,7 +27,7 @@ <h2 class="form__heading">Email settings</h2>
                 }
                 <li>
                     <p>
-                        <a href="@buildIdentityUrl("account/edit")" class="u-underline" data-link-name="identity : email : update">Change your email address</a>  (via Account & Privacy tab)
+                        <a href="@buildIdentityUrl("account/edit")" class="u-underline" data-link-name="identity : email : update">Change your email address</a>
```

Comment:
Bad merge?

Reply:
Nope the tab has not been renamed, the ticket to rename this link was not actually ready to pick up so reverting it.